[flutter_adaptive_scaffold] : 🐛 [FIX] : Issue: 121392. (#3297)
flutter/packages: packages/flutter_adaptive_scaffold/CHANGELOG.md

## 0.1.1

> * 🐛 : FIX - Navigation Rail items not considering `NavigationRailTheme` values - [121135](https://github.com/flutter/flutter/issues/121135)

Reviewer: Can you remove the emoji here to format it as the others?

```suggestion
* Fixes `NavigationRail` items not considering `NavigationRailTheme` values - [flutter/flutter#121135](https://github.com/flutter/flutter/issues/121135)
```

> * When `NavigationRailTheme` is provided, it shall that consider that if user has not given theme related data explicitly.

Reviewer: ```suggestion
* When `NavigationRailTheme` is provided, it will use the theme for values that the user has not given explicit theme-related values for.
```

Author: change log updated .
